cli/internal/cloudcmd/upgrade.go

```diff
@@ -112,6 +112,17 @@ func (u *Upgrader) updateMeasurements(ctx context.Context, newMeasurements measu
 		return nil
 	}
 
+	// don't allow potential security downgrades by setting the warnOnly flag to true
+	for k, newM := range newMeasurements {
```

Comment:
good idea, but can't there be a case where we deliberately stop enforcing a PCR (and maybe enforce another one instead)?

Reply:
Use cases would be very small.
Only two I can really see are.
1. A CSP starts doing weird things in a PCR we currently enforce, resulting in that PCR being unstable.
  * We currently don't enforce any CSP specific PCRs with the exception of PCR[0] on GCP
2. Systemd starts doing weird things in a PCR we currently enforce, resulting in that PCR being unstable.
  * I would trust systemd to not do this suddenly without notice

If any of those 2 cases should happen, we can update the command to allow for changes in specific cases, which would then allow upgrades with a newer CLI version